frontend/src/pages/dashboard.page.tsx

```diff
@@ -2,4 +2,10 @@ import React, { FC } from "react";
 
 import { RouteComponentProps } from "@reach/router";
 
-export const DashboardPage: FC<RouteComponentProps> = () => {};
+type TParams = { id: string };
+
+export const DashboardPage: FC<RouteComponentProps> = (props: RouteComponentProps<TParams>) => {
+  return (
+    <div style={{ margin: "5rem", textAlign: "center" }}>Dashboard</div>

```

Comment:
Here you just added content to the page. Let's move to done the corresponding task on Trello.

Reply:
I need to add content here to fix the error that is showing up